Say deprecated in readme (#2)
viperproject/rust-contracts: README.md

Rust Contracts (Deprecated)
===========================

> **This project needs some work. If you want something more complete, please see Prusti and its contracts library: https://github.com/viperproject/prusti-dev**

Reviewer: "some work" is very generic. What about the following?
```suggestion
**This project is not actively maintained. If you want something that is maintained, please see Prusti and its contracts library: https://github.com/viperproject/prusti-dev**
```
